rotkehlchen/db/utils.py

```diff
@@ -151,27 +151,27 @@ class DBAssetBalance(NamedTuple):
     category: BalanceType
     time: Timestamp
     asset: Asset
-    amount: str
-    usd_value: str
+    amount: FVal
+    usd_value: FVal
 
     def serialize(
             self,
             export_data: Optional[Tuple[Asset, Price]] = None,
-    ) -> Dict[str, Union[str, int]]:
+    ) -> Dict[str, Union[str, int, FVal]]:
         if export_data:
             return {
                 'timestamp': timestamp_to_date(self.time, '%Y-%m-%d %H:%M:%S'),
                 'category': self.category.serialize(),
                 'asset': str(self.asset),
-                'amount': self.amount,
-                f'{export_data[0].symbol.lower()}_value': str(FVal(self.usd_value) * export_data[1]),  # noqa: 501
+                'amount': str(self.amount),
+                f'{export_data[0].symbol.lower()}_value': FVal(self.usd_value) * export_data[1],
```

Comment:
why did you remove the string constructor from here? Isn't this to serialize? This way you will end up with an FVal here.